02/app.js

```diff
@@ -1,11 +1,18 @@
 const calendarJS = {
-    'ES1': '1997-06',
-    'ES2': '1998-06',
-    'ES3': '1999-12',
-    'ES4': null,
-    'ES5': '2009-12',
-    'ES6': '2015-06',
-    'ES7': '2016-06',
-    'ES8': '2017-06',
-    'ES9': '2018-06',
-}
\ No newline at end of file
+	ES1: '1997-06',
+	ES2: '1998-06',
+	ES3: '1999-12',
+	ES4: null,
+	ES5: '2009-12',
+	ES6: '2015-06',
+	ES7: '2016-06',
+	ES8: '2017-06',
+	ES9: '2018-06',
+};
+
+for (const version in calendarJS) {
+	if (calendarJS[version] === null) {
+		calendarJS[version] = 'nie zostało wydane';
+	}
+	console.log(`${version} wydano w terminie ${calendarJS[version]}`);
```

Comment:
Zabrakło `else`, inaczej ten kod będzie zawsze wykonany, a w naszym przypadku nie o to chodzi.

PS. Marcinie, już po paru zadaniach widzę, że masz sporą wiedzę, ale gubisz się na szczegółach. Mam wrażenie, że chcesz za szybko - to może Cię gubić przy większych projektach. Postaraj się zwolnić, sprawdzić 3 razy i iść dalej dopiero wtedy. To zdecydowanie bezpieczniejsze rozwiązanie ;) Wiem, że nie da sie tak od razu zmienić nawyków, ale trzeba od czegoś zacząć :)